### PR TITLE
DOCS-3970 Move php cli information to guide

### PR DESCRIPTION
**Pull request by kayayarai**
### What does this PR do?
Adds back content removed when PHP setup page was tidied

### Motivation
Conversation with PM, DOCS-3970

### Additional Notes
Just adding back content that was previously in another page.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by labbati — APPROVED** [review]
This looks very good to me, thanks!

File: content/en/tracing/guide/trace-php-cli-scripts.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+  tags: "Documentation"
+  text: "Set up PHP trace collection"
+- link: "/tracing/troubleshooting/"
+  tags: "Documentation"
+  text: "Troubleshooting"
+---
+
+## Short-running CLI scripts
+
+A short-running script typically runs for a few seconds or minutes and the expected behavior is to receive one trace each time the script is executed.
```
Comment (cswatt) [inline content/en/tracing/guide/trace-php-cli-scripts.md:15]:
```suggestion
A short-running script typically runs for a few seconds or minutes. The expected behavior is to receive one trace each time the script is executed.
```

File: content/en/tracing/guide/trace-php-cli-scripts.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+By default, tracing is disabled for PHP scripts that run from the command line. Opt in by setting `DD_TRACE_CLI_ENABLED` to `1`.
+
+```
+$ export DD_TRACE_CLI_ENABLED=1
+# Optionally, set the agent host and port if different from localhost and 8126, respectively
+$ export DD_AGENT_HOST=agent
+$ export DD_TRACE_AGENT_PORT=8126
+```
+
+For example, assume the following `script.php` runs a Curl request:
```
Comment (cswatt) [inline content/en/tracing/guide/trace-php-cli-scripts.md:26]:
```suggestion
For example, assume the following `script.php` runs a cURL request:
```

File: content/en/tracing/guide/trace-php-cli-scripts.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+sleep(1);
+```
+
+Run the script:
+
+```
+$ php script.php
+```
+
+Once run, the trace is generated and sent to the Datadog backend when the script terminates.
```
Comment (cswatt) [inline content/en/tracing/guide/trace-php-cli-scripts.md:42]:
```suggestion
Once the script is run, the trace is generated and sent to the Datadog backend when the script terminates.
```

File: content/en/tracing/guide/trace-php-cli-scripts.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+}
+```
+
+Run the script:
+
+```
+$ php long_running.php
+```
+
+Once run, one trace is generated and sent to the Datadog backend every time a new message is processed.
```
Comment (cswatt) [inline content/en/tracing/guide/trace-php-cli-scripts.md:149]:
```suggestion
Once the script is run, one trace is generated and sent to the Datadog backend every time a new message is processed.
```

**Review by cswatt — CHANGES_REQUESTED** [review]
line edits

**Review by cswatt — APPROVED** [review]

**Pull request merged**